[PL-Doc] Move project across organizations (#11619)
harness/developer-hub: docs/platform/organizations-and-projects/move-projects/move-projects-across-organization.md

1. Navigate to **Projects** at account or organization level
2. Click the **⋮** menu next to your project
3. Select **Move Project**

    <DocImage path={require('../static/project-list-view.png')} width="80%" height="80%" title="Move project modal" />

### Step 2: Select destination organization

1. In the Move Project modal, review the warning about potential impacts

  <DocImage path={require('../static/review-move-project.png')} width="50%" height="50%" title="Move project modal" />

2. Select the destination organization from the dropdown. Only organizations where you have Create Project permission are available.

  <DocImage path={require('../static/select-destination-org.png')} width="50%" height="50%" title="Move project modal" />

3. Click **Move Project** to start the move process

  <DocImage path={require('../static/move-project.png')} width="50%" height="50%" title="Move project modal" />

### Step 3: Confirm Move

1. Review the confirmation dialog showing potential impacts. Type the **Project name** to confirm

2. Click **Confirm Move**

  <DocImage path={require('../static/move-confirm.png')} width="50%" height="50%" title="Move project confirm" />

> Once the move completes, you'll be redirected to the project in its new organization. Follow the [post-move remediation](./pre-move-and-post-move-guide/#post-move-remediation) guide to complete the movement.

Author: updated

        title="View Expired Delegates in Harness"
        width="80%"
        height="40%"
        referrerPolicy="strict-origin-when-cross-origin"
        frameBorder="0"
        allowFullScreen
        ></iframe>

3. Select the destination organization from the dropdown where you want to move the selected project. 

    <DocImage path={require('../static/select-destination-org.png')} width="50%" height="50%" title="Move project modal" />

4. Click **Move Project** to proceed

    <DocImage path={require('../static/move-project.png')} width="50%" height="50%" title="Move project modal" />

### Step 3: Confirm Move

1. Review the confirmation dialog showing potential impacts. Type the **Project identifier** to confirm

2. Click **Confirm Move**

    <DocImage path={require('../static/move-confirm.png')} width="50%" height="50%" title="Move project confirm" />

:::note
While a project is being moved, all project-level access control components—including users, service accounts, user groups, role bindings, resource groups, and roles—are updated asynchronously in the background, which may take some time to complete.
:::

> Once a project is moved, you'll be redirected to the project in its new organization. A banner will appear stating: _This project was recently moved from another organization. Some entities may reference resources that no longer exist._ and you can also view the new destination organization, as shown in the image below.

Reviewer: ```suggestion
Once a project is moved, you'll be redirected to the project overview page in its new organization. A banner will appear stating: **This project was recently moved from another organization. Some entities may reference resources that no longer exist.**
```

<DocImage path={require('../static/move-complete.png')} width="80%" height="80%" title="Move project confirm" />

> Follow the [post-move remediation](./pre-move-and-post-move-guide/#post-move-remediation) guide to verify and update any broken references, and ensure the project functions correctly in its new organization. This guide is not exhaustive; you might need additional steps based on your project setup.

Reviewer: There is not enough space after the screenshot and the text. The text is a general statement but it seems to be related to the image because there is no space.
